app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js

```diff
@@ -24,7 +24,6 @@ const addEthereumChain = {
     rejectApprovalRequestsForOrigin: true,
     setTokenNetworkFilter: true,
     setEnabledNetworks: true,
-    setEnabledNetworksMultichain: true,
```

Comment:
How come this is removed? Is this something we don't expose on this handler now?